lib/roo_on_rails/rack/populate_env_from_jwt.rb

```diff
@@ -7,15 +7,22 @@ module Rack
     class PopulateEnvFromJWT
       UnacceptableKeyError = Class.new(RuntimeError)
       VALID_PREFIXES_KEY = 'VALID_IDENTITY_URL_PREFIXES'.freeze
+      DEFAULT_MAPPED_URLS = {
+        'https://test.deliveroo.co.uk/' => 'http://orderweb.rooenv-staging.io/',
+        'https://deliveroo.co.uk/' => 'http://orderweb.deliverooapp.com/',
+        'https://identity-staging.deliveroo.com/' => 'http://identity.rooenv-staging.io/',
```

Comment:
This should be `https://internal-identity.rooenv-staging.io/` as we run a separate "pure internal" instance that doesn't go through any of the publicly exposed load balancers. Same for production that it should be `https://internal-identity.deliveroo.com`.

Note that I'm fairly sure all URLs can be https (with the "s") as that seems to work internally, at least for the URLs I've tried. That would make things much safer if it does work, as the integrity of the key download process is critical to security.